frontend/src/components/DebugInformation/DebugInformation.tsx

```diff
@@ -189,68 +188,79 @@ const DebugInformationComponent: React.FC<DebugInformationProps> = (props: Debug
         }
       }
     }
-
     return info;
   };
 
-  let debugInformation: DebugInformationData = {
-    backendConfigs: {
-      authenticationConfig: authenticationConfig,
-      computedServerConfig: serverConfig
-    },
-    currentURL: window.location.href,
-    reduxState: props.appState
+  const renderDebugInformation = () => {
+    let debugInformation: DebugInformationData = {
+      backendConfigs: {
+        authenticationConfig: authenticationConfig,
+        computedServerConfig: serverConfig
+      },
+      currentURL: window.location.href,
+      reduxState: props.appState
+    };
+    debugInformation = filterDebugInformation(debugInformation);
+    return beautify(debugInformation, parseConfig, 2);
   };
 
-  debugInformation = filterDebugInformation(debugInformation);
-
-  const debugInformationText = beautify(debugInformation, parseConfig, 2);
-
-  const copyText = currentTab === 'kialiConfig' ? JSON.stringify(config, null, 2) : debugInformationText;
+  const getCopyText = (): string => {
+    const text = currentTab === 'kialiConfig' ? JSON.stringify(config, null, 2) : renderDebugInformation();
+    return text;
+  };
 
-  const columns: ThProps[] = [{ title: 'Configuration' }, { title: 'Value' }];
+  const columns = (): ICell[] => {
+    return [{ title: $t('Configuration') }, { title: $t('Value') }];
+  };
 
-  let rows: IRow[] = [];
+  const getRows = () => {
+    var conf: string[][] = [];
 
-  for (const [k, v] of Object.entries(config)) {
-    if (typeof v !== 'string') {
-      rows.push({ cells: [k, JSON.stringify(v)] });
-    } else {
-      rows.push({ cells: [k, v] });
+    for (const [k, v] of Object.entries(config)) {
+      if (typeof v !== 'string') {
+        conf.push([k, JSON.stringify(v)]);
+      } else {
+        conf.push([k, v]);
+      }
     }
-  }
+    return conf;
+  };
 
-  const renderTabs = (): React.ReactNode[] => {
+  const renderTabs = () => {
     const kialiConfig = (
-      <Tab eventKey={0} title="Kiali Config" key="kialiConfig">
-        <CopyToClipboard onCopy={copyCallback} text={rows} options={copyToClipboardOptions}>
-          <SimpleTable label="Debug Information" className={tableStyle} columns={columns} rows={rows} />
+      <Tab eventKey={0} title={$t('KialiConfig', 'Kiali Config')} key="kialiConfig">
```

Comment:
Please let SimpleTable component I don't know why we need this change

Reply:
I just looked at the code, and it seems like I just changed the title, and the rest of the code is still there. Is this a bug in the review?